src/App.tsx

```diff
@@ -1,109 +1,39 @@
 import React from 'react';
 
 import './App.scss';
+import { Post } from './types/Post';
 
```

Comment:
remove unnecessary lines
```suggestion
import './App.scss';
import { Post } from './types/Post';
```